package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "react-streamfield",
+  "version": "0.6.10",
+  "author": "NoriPyt <contact@noripyt.com> (https://noripyt.com)",
+  "description": "Powerful field for inserting multiple blocks with nesting.",
+  "keywords": [
+    "react",
+    "react-component",
+    "field"
+  ],
+  "homepage": "https://github.com/noripyt/react-streamfield",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/noripyt/react-streamfield"
+  },
+  "bugs": {
+    "url": "https://github.com/noripyt/react-streamfield/issues"
+  },
+  "license": "BSD",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "main": "src/index.js",
+  "devDependencies": {
+    "@babel/cli": "7.1.5",
+    "@babel/core": "7.1.6",
+    "@babel/plugin-proposal-class-properties": "7.1.0",
+    "@babel/plugin-proposal-decorators": "7.1.6",
+    "@babel/plugin-proposal-object-rest-spread": "7.0.0",
+    "@babel/preset-env": "7.1.6",
+    "@babel/preset-react": "7.0.0",
+    "babel-core": "7.0.0-bridge.0",
+    "babel-loader": "8.0.4",
+    "babel-plugin-transform-react-remove-prop-types": "0.4.21",
+    "classnames": "2.2.6",
```

Comment:
This should be in `dependencies`, as well as: `classnames`, `react-animate-height`, `react-beautiful-dnd`, `uuid` (`react-redux`, `redux`, `redux-thunk`, `react`, `react-dom`, `prop-types`)